### PR TITLE
ELEMENTS-1416: trigger search in results view when clicking the clear button

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

**Comment by richardsd** [comment]
Legacy pipeline here https://qa.nuxeo.org/jenkins/job/2021/job/web-ui-pipeline/91/ ⏳

File: ui/search/nuxeo-results-view.js [hunk 1]
```diff
@@ -399,9 +399,10 @@ import './nuxeo-search-results-layout.js';
         if (evt.detail.path) {
           const param = evt.detail.path.split('.')[1];
           this.notifyPath(`_params.${param}`, evt.detail.value);
-          if (this.visible && this.auto) {
-            this._search();
-          }
+        }
+        // search should happen even if e.detail doesn't exist, because we are clearing the form
+        if (this.visible && this.auto) {
```
Comment (Gabez0r) [inline ui/search/nuxeo-results-view.js]: Are we sure that the only situation in which `evt.detail.path` evaluates to false is when we are clearing the form?

Comment (richardsd) [reply ui/search/nuxeo-results-view.js]: From all the tests and did, I would say yes. But in fact, I don't really know. I opened the PR with this solution to check the impact of the change (and run all the ftests). I might ping you directly for some discussion about the topic. FYI: we have this behavior in 10.10 too.

**Review by Gabez0r — APPROVED** [review]

**Comment by richardsd** [comment]
Legacy pipeline https://qa.nuxeo.org/jenkins/job/2021/job/web-ui-pipeline/116/ 🔵

**Review by mnixo — APPROVED** [review]

**Pull request merged**